### DISCUSSION TITLE
Node-red and RasberryPi

### DISCUSSION THREAD

**Discussion by Malvin76** [Yogui79/IntexPureSpa#46] [Q&A]
Hello all,
Would it be possible to transfer these information in a node-red node to use it for a rasberry PI?
Can someone do this?

Thanks.
Cheers Peter

**Comment by LEECHER1** [comment]
Why do you want it with Raspberry PI?
The Raspberry ist to strong for it. 

The ESP32 have all to you want and ist smaller, the consumption is low and it is easier.

**Reply by Malvin76** [reply]
Because... i managed my Swimmingpool stuff with the PI and the Node-red and Now i want to implement this Whirlpool in my Overall Dashboard..

**Reply by w00dst0ck** [reply]
You could connect node-red to mqtt .
